README.md

```diff
@@ -4,24 +4,27 @@ Various experiments using the python pymc3 library for Bayesian learning.
 
 ## Quick start
 
-Install the virtual environment `posteriorpymc3`:
+### Installing `uv`
 
-### Windows
+The Python project manager selected is `uv`. To install the project 
+dependencies, run the following command:
 
 ```bash
-> py -m pip install --upgrade pip
-> py -m pip install --user virtualenv
-> py -m venv posteriorpymc3
-> .\posteriorpymc3\Scripts\activate
-(posteriorpymc3) > pip install -r requirements.txt
+pip install uv
+uv sync
 ```
 
-### Unix/macOS
+### Project management
 
-```bash
-> py -m pip install --upgrade pip
-> py -m pip install --user virtualenv
-> py -m venv posteriorpymc3
-. source posteriorpymc3/bin/activate
-> pip install -r requirements.txt
+`uv` is a project manager that works in a similar way to `poetry`. 
+For this reason, the commands are similar. For instance, you can add or 
```

Comment:
This line seems redundant:
`For this reason, the commands are similar.`

Reply:
That's true, I deleted that line.

Reply:
I can still see the line in youtr branch, please push your changes.

Reply:
Done!